jupyter_client/kernelspec.py

```diff
@@ -195,24 +163,10 @@ def install_kernel_spec(self, source_dir, kernel_name=None, user=False,
         shutil.copytree(source_dir, destination)
 
     def install_native_kernel_spec(self, user=False):
-        """Install the native kernel spec to the filesystem
-
-        This allows a Python 3 frontend to use a Python 2 kernel, or vice versa.
-        The kernelspec will be written pointing to the Python executable on
-        which this is run.
+        """DEPRECATED: Use ipython_kernel.kenelspec.install"""
```

Comment:
Should it warn? I guess it's unlikely that anyone is calling this directly, but maybe it's best for consistency as we move things around.